src/main/java/build/buildfarm/instance/shard/RedisShardSubscription.java

```diff
@@ -87,14 +87,17 @@ private void mainLoop() throws IOException {
     }
   }
 
-  public void stop() {
+  public void stop() throws JedisException {
```

Comment:
JedisException is unchecked and unnecessary as a declaration here.

Reply:
OK, I'll remove